academy/4-my-own-chain/events.md

```diff
@@ -62,7 +62,7 @@ ctx.EventManager().EmitEvent(
 
 The only thing left to do is to implement this correspondingly in the GUI or include a server to listen for such events.
 
-## Player moved event
+## Player moved the event
```

Comment:
```suggestion
## Player-moved event
```
It is an event about the player having made a move. The player did not move any event.